pkg/kp/kv.go

```diff
@@ -79,14 +81,29 @@ type WatchResult struct {
 	Expires time.Time `json:"Expires,omitempty"`
 }
 
+func normalizeOutput(in string) interface{} {
+	var obj map[string]interface{}
+	if json.Unmarshal([]byte(in), &obj) != nil {
+		return in
+	}
+	for key := range obj {
+		if strings.ToLower(key) == "time" {
```

Comment:
For a subset of hosts in our infrastructure, this normalization will allow the service to have an average write rate of 0.  With just the rate limiter, they will write once every 15 seconds.

With the way our infrastructure has been behaving, I anticipate that it's worthwhile to reduce the write workload whenever possible.  But is it _necessary_?  We'll see what happens when I roll out this change ;)